backend/src/main/java/com/huellapositiva/domain/actions/UploadCurriculumVitaeAction.java

```diff
@@ -20,14 +19,21 @@
 
     private final RemoteStorageService remoteStorageService;
 
-    private final VolunteerService volunteerService;
-
     private final VolunteerRepository volunteerRepository;
 
+    /**
+     * This method uploads a file which contains the volunteer resumé (CV) and links its URL to the volunteer
+     *
+     * @param cv
+     * @param volunteerEmail
+     * @throws IOException when the cv is corrupted
+     */
     public void execute(MultipartFile cv, String volunteerEmail) throws IOException {
-        Volunteer volunteer = volunteerService.findVolunteerByEmail(volunteerEmail);
-        URL cvUrl = remoteStorageService.uploadVolunteerCV(cv, volunteer.getId().toString());
-        volunteer.setCurriculumVitae(cvUrl);
-        volunteerRepository.updateCurriculumVitae(volunteer);
+        if (cv != null) {
```

Comment:
If cv is null this action does nothing and succeeds. If I want to upload a cv and there is no cv, I would rather fail the request with a 400

Reply:
Changed. Now in this action file we are checking if the file comes empty or if it is not a file attached. In that case, we fail the request with a 400 error.